### PR TITLE
[CUR-213] Add help center banner to curric visualiser

### PR DESCRIPTION
**Pull request by sonalisin**
## Description

- Added help centre banner to bottom of curriculum visualiser page
- Includes button that redirects to help centre

## How to test

1. Go to https://deploy-preview-1935--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Click on any subject phase combination
3. Click on the curriculum visualiser
4. You should see the banner at the bottom of the page
5. Click the button as a link to test it redirects correctly to the help centre

## Screenshots

How it used to look (delete if n/a):
Screenshot 2023-09-27 at 11 49 08

How it should now look:
Screenshot 2023-09-27 at 16 57 02

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Comment by rightsaidjames** [comment]
Issues found (ignoring placeholder icon):
* Semantically, the `Need help with...` heading should be a H2 heading element, so it's on a par with the year group headings rather than subordinate to the last year group heading (year 11 in the example below).
* The gap between the heading and the body text should be larger.
* The body text doesn't appear to have a text size defined (which means that it will default to the browser default) - it should be 18px with a line height of 28px. 
* Semantically, the body text should also be a paragraph element rather than a div.
* The gap between the bottom of the last year group container (pink area) and the banner should be 48px, but it's only 36px.

Screenshot 2023-09-27 at 15 16 14

Screenshot 2023-09-27 at 15 31 18

**Comment by HelenHarlow** [comment]
Couple of things in addition to James’ comments above:
* On tablet, there seems to be some overlap with the copy and the button. Could we have a minimum 24px gap between? Just for the smaller tablet screens where that overlap occurs
* On mobile, I had designed a stacked layout (see screenshot) which works better on smaller screens, if it's possible to implement that?

Screenshot 2023-09-27 at 16 01 42

Screenshot 2023-09-27 at 16 02 20

Screenshot 2023-09-27 at 16 02 58

**Comment by rightsaidjames** [comment]
Apologies, for some reason I forgot to check tablet and mobile views!

Anyway, it all looks good, apart from:
> Semantically, the body text should also be a paragraph element rather than a div.

Which is much less important than the other feedback on this PR, but probably worth fixing.

**Review by rightsaidjames — APPROVED** [review]

**Comment by sonalisin** [comment]
> Apologies, for some reason I forgot to check tablet and mobile views!
> 
> Anyway, it all looks good, apart from:
> 
> > Semantically, the body text should also be a paragraph element rather than a div.
> 
> Which is much less important than the other feedback on this PR, but probably worth fixing.

I did change it from P to Typography, but let me investigate..

File: src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx [hunk 1]
```diff
@@ -248,269 +250,323 @@ const UnitsTab: FC<UnitsTabProps> = ({ data }) => {
...
+              name="books"
+              size={92}
+              $background={"teachersRed"}
+              $mr={40}
+              $mb={[24, 0]}
+              $color={"black"}
+            />
+
+            <Flex
+              $width={"70%"}
```
Comment (k-huggs) [inline src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx:545]: This 70% width is preventing the text from stretching on phone view, might need to alter it to 100% on phone view

**Review by k-huggs — APPROVED** [review]
LGTM ✅

**Comment by HelenHarlow** [comment]
I'm not sure if this is related to this ticket or not but it looks like now on the overview tab, there is no spacing between the partner bio and footer. Is it possible to add in 48px spacing?
Screenshot 2023-09-28 at 10 52 10

**Comment by rightsaidjames** [comment]
This was an unintended change resulting from changes to the footer made by the lesson squad, also affecting the Curriculum landing  page:
Screenshot 2023-09-28 at 11 57 57

So we could probably fix it is part of that ticket (if that's okay with @sonalisin), but it will need fixing in both places. 

FYI @jack-skerrett-oak

**Review by HelenHarlow — APPROVED** [review]
Looks good!

**Pull request merged**